core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockStore.java

```diff
@@ -78,14 +86,65 @@ public class RocksBlockStore implements BlockStore {
   public RocksBlockStore(String baseDir) {
     RocksDB.loadLibrary();
     mDisableWAL = new WriteOptions().setDisableWAL(true);
-    mIteratorOption = new ReadOptions().setReadaheadSize(
-        ServerConfiguration.getBytes(PropertyKey.MASTER_METASTORE_ITERATOR_READAHEAD_SIZE));
-    mColumnFamilyOptions = new ColumnFamilyOptions()
-        .setMemTableConfig(new HashLinkedListMemTableConfig())
-        .setCompressionType(CompressionType.NO_COMPRESSION);
-    List<ColumnFamilyDescriptor> columns = Arrays.asList(
-        new ColumnFamilyDescriptor(BLOCK_META_COLUMN.getBytes(), mColumnFamilyOptions),
-        new ColumnFamilyDescriptor(BLOCK_LOCATIONS_COLUMN.getBytes(), mColumnFamilyOptions));
+    mReadPrefixSameAsStart = new ReadOptions().setPrefixSameAsStart(true);
+    mIteratorOption = new ReadOptions()
```

Comment:
Move these assignments (and `mDisableWAL` above) to field declaration.

Reply:
These have to be created after RocksDB.loadLibrary is called, also for now throwing the exceptions are ok here. I made a note about it in https://tachyonnexus.atlassian.net/browse/AE-2460 which is suggested to use Dependency Injection to improve this code.